src/main/java/me/jellysquid/mods/lithium/common/util/streams/ConcatSpliterator.java

```diff
@@ -0,0 +1,117 @@
+package me.jellysquid.mods.lithium.common.util.streams;
+
+import java.util.Comparator;
+import java.util.Spliterator;
+import java.util.function.Consumer;
+
+abstract class ConcatSpliterator<T, S extends Spliterator<T>>
```

Comment:
This code is taken from a library with a license that is incompatible with ours, isn't it?

Reply:
No this is slightly modified code from OpenJDK which is licensed under the GPL license and is therefore compatible with the LGPL.

Reply:
OpenJDK is licensed under GPLv2 which is incompatible with this projects LGPLv3: https://www.gnu.org/licenses/gpl-faq.html#AllCompatibility